RotationControl/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    def packageName = 'com.programminghoch10.RotationControl'
+    namespace packageName
+    
+    defaultConfig {
+        applicationId packageName
+        minSdk 29
```

Comment:
need to check how low this can go,
there are checks for sdk lower than 29 present in the code already